CITATION.cff

```diff
@@ -0,0 +1,47 @@
+cff-version: 1.2.0
+
+title: "mr_proc"
+
+version: 0.1.0
+
+abstract: A workflow manager for curating MRI and tabular data and standardized processing. 
+
+message: "If you use this software, please cite it as below."
+
+repository-code: "https://github.com/neurodatascience/mr_proc.git"
+
+
+contact:
+  - affiliation: "ORIGAMI lab, McGill University, Montréal, Canada"
+    email:  nikhil.bhagwat@mcgill.ca
+    family-names: Bhagwat
+    given-names: Nikhil
+
+authors:
+  - affiliation: "ORIGAMI lab, McGill University, Montréal, Canada"
+    email:  nikhil.bhagwat@mcgill.ca
+    family-names: Bhagwat
+    given-names: Nikhil
+    orcid: "https://orcid.org/0000-0002-5276-8818"    
```

Comment:
This belongs to some `Alice Van Audenhaege`

Reply:
This is me:
`0000-0001-6073-7141`

Reply:
should be fixed now